arex-storage-web-api/src/main/java/com/arextest/storage/repository/scenepool/ScenePoolProviderImpl.java

```diff
@@ -49,6 +49,14 @@ public void upsertOne(Scene scene) {
     getCollection().updateOne(filter, update, new UpdateOptions().upsert(true));
   }
```

Comment:
updateOne will update all items with the same sceneKey, please check if it meets the requirements.